tools/cldr-apps/src/main/webapp/tc-all-errors.jsp

```diff
@@ -59,7 +59,7 @@ if(request.getParameter("flush") != null) {
 <body>
 
 
-<h1>All error subTypes</h1>
+<h1>All error subtypes</h1>
```

Comment:
this and following, not a bad change, doesn't seem related to dashboard…

Reply:
right -- globally replacing subType with subtype was easier and seems to me safer and better in the long run